{"uuid":"516451ea-3a64-4841-a185-197191b15275","historyId":"1ca168352624e199eccc15a6b90f0d1b","fullName":"com.api.test.AuthenticateTest.authenticatePOSTHeaderTest","labels":[{"name":"package","value":"com.api.test.AuthenticateTest"},{"name":"testClass","value":"com.api.test.AuthenticateTest"},{"name":"testMethod","value":"authenticatePOSTHeaderTest"},{"name":"parentSuite","value":"Default suite"},{"name":"suite","value":"Default test"},{"name":"subSuite","value":"com.api.test.AuthenticateTest"},{"name":"host","value":"MUM16123CPU2825"},{"name":"thread","value":"11332@MUM16123CPU2825.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"},{"name":"feature","value":"User Authentication"},{"name":"epic","value":"Authentication"},{"name":"story","value":"User submits valid credentials (userid and password) for login"},{"name":"severity","value":"normal"}],"links":[],"name":"authenticatePOSTHeaderTest","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"Error during validation:","trace":"java.lang.AssertionError: Error during validation:\r\n\tat org.testng.Assert.fail(Assert.java:87)\r\n\tat com.api.comman.JsonPojoArrayValidator.validateJsonArrayAgainstPojo(JsonPojoArrayValidator.java:96)\r\n\tat com.api.response.mapper.AuthenticateMapper.authenticateResponseValidation(AuthenticateMapper.java:29)\r\n\tat com.api.test.AuthenticateTest.authenticatePOSTHeaderTest(AuthenticateTest.java:47)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\r\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\r\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)\r\n\tat org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)\r\n\tat org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)\r\n\tat org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)\r\n\tat org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)\r\n\tat org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)\r\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)\r\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1541)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:794)\r\n\tat org.testng.TestRunner.run(TestRunner.java:596)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:377)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:276)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1134)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1063)\r\n\tat org.testng.TestNG.run(TestNG.java:1031)\r\n\tat org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)\r\n\tat org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)\r\n\tat org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)\r\nCaused by: com.fasterxml.jackson.databind.exc.MismatchedInputException: Cannot deserialize value of type `java.util.ArrayList<java.lang.Object>` from Object value (token `JsonToken.START_OBJECT`)\n at [Source: (String)\"{\"student\":{\"certificatePrograms\":null,\"sapid\":\"77221782653\",\"sem\":null,\"email\":null,\"lastName\":\"Sen\",\"firstName\":\"Shekhar\",\"middleName\":\"\",\"fatherName\":\"Shankar\",\"husbandName\":\"\",\"motherName\":\"Chinu\",\"gender\":\"Male\",\"program\":\"BBA\",\"enrollmentMonth\":\"Jan\",\"enrollmentYear\":\"2022\",\"emailId\":\"jollylancher100@gmail.com\",\"mobile\":\"8011517262\",\"altPhone\":\"\",\"dob\":\"2002-09-25\",\"regDate\":\"2022-02-21\",\"isLateral\":\"N\",\"isReReg\":\"\",\"address\":\"sen bakery, Bordumsa,Bordumsa bazar ,786174,Tinsukia\",\"houseNoN\"[truncated 7560 chars]; line: 1, column: 1]\r\n\tat com.fasterxml.jackson.databind.exc.MismatchedInputException.from(MismatchedInputException.java:59)\r\n\tat com.fasterxml.jackson.databind.DeserializationContext.reportInputMismatch(DeserializationContext.java:1601)\r\n\tat com.fasterxml.jackson.databind.DeserializationContext.handleUnexpectedToken(DeserializationContext.java:1375)\r\n\tat com.fasterxml.jackson.databind.DeserializationContext.handleUnexpectedToken(DeserializationContext.java:1322)\r\n\tat com.fasterxml.jackson.databind.deser.std.CollectionDeserializer.handleNonArray(CollectionDeserializer.java:392)\r\n\tat com.fasterxml.jackson.databind.deser.std.CollectionDeserializer.deserialize(CollectionDeserializer.java:252)\r\n\tat com.fasterxml.jackson.databind.deser.std.CollectionDeserializer.deserialize(CollectionDeserializer.java:28)\r\n\tat com.fasterxml.jackson.databind.deser.DefaultDeserializationContext.readRootValue(DefaultDeserializationContext.java:322)\r\n\tat com.fasterxml.jackson.databind.ObjectMapper._readMapAndClose(ObjectMapper.java:4593)\r\n\tat com.fasterxml.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:3548)\r\n\tat com.fasterxml.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:3516)\r\n\tat com.api.comman.JsonPojoArrayValidator.validateJsonArrayAgainstPojo(JsonPojoArrayValidator.java:30)\r\n\t... 30 more\r\n"},"stage":"finished","description":"Send a POST request to the authentication URL and validate the response","steps":[],"attachments":[],"parameters":[],"start":1732164671430,"stop":1732164672795}